spec/models/scheduled_message_spec.rb

```diff
@@ -12,4 +12,35 @@
      it { should validate_presence_of(:time_of_day) }
      it { should validate_presence_of(:title) }
   end
+
+  describe '.date_time_ordering' do
+    let(:tags) { ['testing', 'testing2'] }
+    let!(:message1) do
+      ScheduledMessage.create!(title: 'test1',
+                               body: 'this is the first test',
+                               days_after_start: 1,
+                               time_of_day: '2000-01-01 16:00:00 UTC',
+                               tag_list: tags)
+    end
+
+    let!(:message2) do
+      ScheduledMessage.create!(title: 'test2',
+                               body: 'this is the second test',
+                               days_after_start: 1,
+                               time_of_day: '2000-01-01 14:00:00 UTC',
+                               tag_list: tags)
+    end
+
+    let!(:message3) do
+      ScheduledMessage.create!(title: 'test3',
+                               body: 'this is the third test',
+                               days_after_start: 1,
+                               time_of_day: '2000-01-01 11:00:00 UTC',
+                               tag_list: tags)
+    end
+
+    it 'should display the messages in chronological order' do
+      expect(ScheduledMessage.date_time_ordering.first).to eq(message3)
```

Comment:
how about using `match_array` here rather than just looking at the first element? that way we know the ordering is working for all items.

also, we are mostly not using `let` statements in this test suite. some explanation of why: https://robots.thoughtbot.com/lets-not

so can we create the scheduled messages in the test itself rather than in `let` statements? Using the `scheduled_message` factory would be great, too.